app/views/Overlap.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
What's this for?

Reply:
Ah, sorry, this was for personal sanity editing the file - with Prettier enabled, it's just a mess of red lines. Didn't want to do a Prettier fix in the same PR so temporarily disabled it so I could edit. Will remove the line